elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/AggregationTest.java

```diff
@@ -401,6 +401,34 @@ public static void setupInstance() throws Exception {
             "aggregations:{'v1.max.field': 'val1'",
             "'v2.min.field': 'val2'}"))
         .returnsUnordered("v1=7; v2=5");
+  }
+
+  @Test void testMultiNullsSortInGroup() {
+    CalciteAssert.that()
+        .with(newConnectionFactory())
```

Comment:
According the query  and result, Actually I can't judge the result is wrong or right. So please make the test data more clearly for your test.

Reply:
@NobiGo SortTest has been added and its results are more intuitive.